src/TorchSharp/NN/Linear.cs

```diff
@@ -70,12 +74,9 @@ public static partial class nn
             /// <param name="hasBias">If set to false, the layer will not learn an additive bias.</param>
             /// <param name="device">The desired device of the parameters and buffers in this module</param>
             /// <param name="dtype">The desired floating point or complex dtype of the parameters and buffers in this module</param>
-            public static Linear Linear(long inputSize, long outputSize, bool hasBias = true, Device? device = null, ScalarType? dtype = null)
+            public static Linear Linear(long inputSize, long outputSize, bool hasBias = true, Device device = null, ScalarType? dtype = null)
             {
-                var res = THSNN_Linear_ctor(inputSize, outputSize, hasBias, out var boxedHandle);
```

Comment:
Is this going to be deleted from https://github.com/dotnet/TorchSharp/blob/main/src/Native/LibTorchSharp/THSNN.cpp ?